source/ethereum-clients/cpp-ethereum/building-from-source/linux-fedora.rst

```diff
@@ -3,117 +3,16 @@
 Installing dependencies for Fedora
 ################################################################################
 
-Fedora 20
+Fedora 24
 --------------------------------------------------------------------------------
 Steps: ::
-
-    yum install git automake autoconf libtool cmake gcc gcc-c++ xkeyboard-config \
-            leveldb-devel boost-devel gmp-devel cryptopp-devel miniupnpc-devel \
-            qt5-qtbase-devel qt5-qtdeclarative-devel qt5-qtquick1-devel qt5-qtwebkit-devel \
-            mesa-dri-drivers snappy-devel ncurses-devel readline-devel curl-devel \
-            python-devel
-
-Fedora 21
---------------------------------------------------------------------------------
-Steps: ::
-
-    yum install git automake autoconf libtool cmake gcc gcc-c++ xkeyboard-config \
+  
+    dnf install git automake autoconf libtool cmake gcc gcc-c++ xkeyboard-config \
```

Comment:
You probably only need: 

`git cmake gcc-c++ boost-devel leveldb-devel curl-devel libmicrohttpd-devel gmp-devel`